pkg/reconciler/events/cloudevent/cloudevent.go

```diff
@@ -88,8 +88,15 @@ func (t TektonEventType) String() string {
 	return string(t)
 }
 
-// CEClient matches the `Client` interface from github.com/cloudevents/sdk-go/v2/cloudevents
-type CEClient cloudevents.Client
+// CEClient wraps the `Client` interface from github.com/cloudevents/sdk-go/v2/cloudevents
+// and has methods to count the cloud events being sent
```

Comment:
NIT: perhaps it's worth commenting that the extra methods are for testing purposes only

Reply:
Sure!